### ISSUE TITLE
[Bug]: worldguard permissions issue

### ISSUE THREAD

**Issue by A-Kevin1217** [spartacus04/jext-reborn#331]
### Describe the bug

I don't have the right to play on the record player for a limited time.
I didn't modify any permission configuration.

### Steps to reproduce

1. Take out the record. 2. Right-click the record player. 3. Can't put it in. 4. Squat down and put it in. 5. The record sound is the sound of record 11.

### Expected behaviour

I hope it can play normally.

### Server Version

1.20.4

### Server Type

Paper

### Jukebox Behaviour

Vanilla

### Before submitting the issue

- [X] I have searched the issue tracker and did not find an issue describing my bug.
- [X] I agree to the fact that if this issue becomes inactive it will be closed.
- [X] I agree to the fact that if most fields in this issue are not filled correctly the issue will be marked as invalid and closed.

**Comment by A-Kevin1217** [comment]
![image](https://github.com/spartacus04/jext-reborn/assets/21151999/b9231fb7-65c4-49d5-a207-aa0c69aa7fa5)

**Comment by spartacus04** [comment]
Hello! Thanks for the bug report. I've noticed you have worldguard installed in your server, perhaps that's causing issues? Are you using the plugin in the common region or in another region?

**Comment by A-Kevin1217** [comment]
> Hello! Thanks for the bug report. I've noticed you have worldguard installed in your server, perhaps that's causing issues? Are you using the plugin in the common region or in another region?

maybe common？I can tell you my servers-ip，you need it ？

**Comment by A-Kevin1217** [comment]
> Hello! Thanks for the bug report. I've noticed you have worldguard installed in your server, perhaps that's causing issues? Are you using the plugin in the common region or in another region?

If you have op permission, you can use the record player.

**Comment by A-Kevin1217** [comment]
> Hello! Thanks for the bug report. I've noticed you have worldguard installed in your server, perhaps that's causing issues? Are you using the plugin in the common region or in another region?

Sure, it is caused by worldguard, but how can I solve it and make them coexist?

**Comment by A-Kevin1217** [comment]
> I'll try to look into it this evening as right now I can't really test this. Another person reported a bug with worldguard recently but they closed their issue themselves so I really don't know how they solved it.

Can you quote the serial number of issues? I want to ask.

**Comment by spartacus04** [comment]
#329

**Comment by A-Kevin1217** [comment]
> #329 

thanks

**Comment by gavinlpicard** [comment]
Unfortunately I am also having this issue. Removing World Guard is not an option for me. Someone appears to have sent this in the wrong place but it seems they may have determined the cause of the issue:
https://github.com/spartacus04/jext-reborn/issues/287#issuecomment-2057979149

**Comment by spartacus04** [comment]
> Unfortunately I am also having this issue. Removing World Guard is not an option for me. Someone appears to have sent this in the wrong place but it seems they may have determined the cause of the issue: [#287 (comment)](https://github.com/spartacus04/jext-reborn/issues/287#issuecomment-2057979149)

Yeah, they made a pr "fixing" this, however i think it just broke with their pr. I'll try to test the old method I was doing it

**Comment by spartacus04** [comment]
Could you perhaps try if the `2.0.3` update works with worldguard? if that's the case i'll revert the changes

**Comment by A-Kevin1217** [comment]
> Could you perhaps try if the 2.0.3 update works with worldguard? if that's the case i'll revert the changes

I have tried 2.0.3, and it seems that there is still the same problem.

**Comment by spartacus04** [comment]
fixed in 2.1
